Tools/_sunrise/changelog/changelog_targets.py

```
@@ -0,0 +1,116 @@
+
+
+@dataclass(frozen=True)
+class ChangelogTarget:
+    target_id: str
+    changelog_file: Path
+    webhook_secret: str
+
+
+def validate_target_id(value: str | None) -> str:
+    if not isinstance(value, str) or not value or "\n" in value or "\r" in value:
+        raise RuntimeError("Идентификатор цели не может быть пустым или содержать перенос строки")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)